distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/JvmOptionsParser.java

```diff
@@ -95,7 +95,16 @@ public static List<String> determineJvmOptions(
 
         try {
             return Collections.unmodifiableList(
-                parser.jvmOptions(args, args.configDir(), tmpDir, envOptions, substitutions, processInfo.sysprops(), machineDependentHeap)
+                parser.jvmOptions(
+                    args,
+                    args.configDir(),
+                    processInfo.workingDir(),
```

Comment:
Spotless makes it hard to see what I changed here. I added `processInfo.workingDir()`.